doc/user_guide/visualisation.rst

```diff
@@ -50,12 +50,21 @@ the x-axis if 1D:
 To change the current coordinates, click on the pointer (which will be a line
 or a square depending on the dimensions of the data) and drag it around. It is
 also possible to move the pointer by using the numpad arrows **when numlock is
-on and the spectrum or navigator figure is selected**.When using the numpad
+on and the spectrum or navigator figure is selected**. When using the numpad
 arrows the PageUp and PageDown keys change the size of the step.
 
 An extra cursor can be added by pressing the ``e`` key. Pressing ``e`` once
 more will disable the extra cursor:
 
+In matplotlib, left and right arrow keys are by default set to navigate the 
+"zoom" history. To avoid the problem of changing zoom while navigating, 
+Ctrl + arrows can be used instead. 
+
+To navigate navigation dimensions larger than 2, modifier keys can be used.
+Ctrl + Shift + left/right, Ctrl + Shift + up/down, Ctrl + Alt + left/right 
```

Comment:
Ctrl + Shift + left/right and Ctrl + Shift + up/down (Ctrl + Alt + left/right and Ctrl + Alt + up/down) navigates dimensions 2 and 3 (4 and 5) respectively?

Reply:
Yes, I like that better.